RashakDude/Project Euler/problem9.py

```diff
@@ -1,6 +1,6 @@
-for a in range(1,1001):

```

Comment:
You are not allowed to make changes in the files of other people :)
Even though there is not really a change, you are creating a new diff.
Better option, fork again. Make PR from a new branch and make readme changes there.